<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>638</width>
    <height>429</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>0</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 20px;</string>
    </property>
    <property name="text">
     <string>Consultorio Odontologico</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 15px;</string>
    </property>
    <property name="text">
     <string>Paciente:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size:15px;</string>
    </property>
    <property name="text">
     <string>Lista Pacientes</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>100</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size:15px;</string>
    </property>
    <property name="text">
     <string>Em atendimento</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LblNome">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>60</y>
      <width>251</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="BtnAdc">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>60</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Adicionar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BtnPass">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>200</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BtnFechar">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>360</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Fechar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BtnLimpar">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>330</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Limpar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BtnMostrar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Mostrar Vetor</string>
    </property>
   </widget>
   <widget class="QListWidget" name="LblPac">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="LblAt">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>120</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
    <property name="sortingEnabled">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>638</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
